Xabe.FFmpeg/Conversion/Conversion.cs

```diff
@@ -174,6 +193,26 @@ public IConversion SetSeek(TimeSpan? seek)
             return this;
         }
 
+        /// <inheritdoc />
+        public IConversion SetInputTime(TimeSpan? time)
+        {
+            if (time.HasValue)
+            {
+                _inputTime = $"-t {time.Value.ToFFmpeg()} ";
+            }
+            return this;
+        }
+
+        /// <inheritdoc />
+        public IConversion SetOutputTime(TimeSpan? time)
+        {
+            if (time.HasValue)
+            {
+                _outputTime = $"-t {time.Value.ToFFmpeg()} ";
```

Comment:
You're setting this but never use in conversion

Reply:
I have added the usage of these into the conversion and added tests.